2025/01/18 03:04:32 [error] 277#277: *32 [lua] lets_encrypt.lua:40: issue_cert(): auto-ssl: dehydrated failed: env HOOK_SECRET=d62814ab93e83aeed96c2e9d62bf359d5aca294c78eae74daa542170c4d008f7 HOOK_SERVER_PORT=8999 /usr/local/bin/resty-auto-ssl/dehydrated --cron --accept-terms --no-lock --domain 179.232.255.135 --challenge http-01 --config /etc/resty-auto-ssl/letsencrypt/config --hook /usr/local/bin/resty-auto-ssl/letsencrypt_hooks status: 256 out: # INFO: Using main config file /etc/resty-auto-ssl/letsencrypt/config
startup_hook
Processing 179.232.255.135
 + Signing domains...
 + Generating private key...
 + Generating signing request...
 + Requesting new certificate order from CA...
 err:   + ERROR: An error occurred while sending post-request to https://acme-v02.api.letsencrypt.org/acme/new-order (Status 400)

Details:
HTTP/2 400 
server: nginx
date: Sat, 18 Jan 2025 06:04:31 GMT
content-type: application/problem+json
content-length: 226
boulder-requester: 2176686025
cache-control: public, max-age=0, no-cache
link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
replay-nonce: TuPv3OuuTUNeVfx-HUea_uxHzS1hY9bOHwAw9jf3ksJnGv4pR0s

{
  "type": "urn:ietf:params:acme:error:rejectedIdentifier",
  "detail": "Invalid identifiers requested :: Cannot issue for \"179.232.255.135\": The ACME server can not issue a certificate for an IP address",
  "status": 400
}

, context: ssl_certificate_by_lua*, client: 172.27.80.1, server: 0.0.0.0:443
2025/01/18 03:04:32 [error] 277#277: *32 [lua] ssl_certificate.lua:97: issue_cert(): auto-ssl: issuing new certificate failed: dehydrated failure, context: ssl_certificate_by_lua*, client: 172.27.80.1, server: 0.0.0.0:443
2025/01/18 03:04:32 [error] 277#277: *32 [lua] ssl_certificate.lua:291: auto-ssl: could not get certificate for 179.232.255.135 - using fallback - failed to get or issue certificate, context: ssl_certificate_by_lua*, client: 172.27.80.1, server: 0.0.0.0:443
2025/01/18 03:04:42 [error] 277#277: *46 [lua] lets_encrypt.lua:40: issue_cert(): auto-ssl: dehydrated failed: env HOOK_SECRET=d62814ab93e83aeed96c2e9d62bf359d5aca294c78eae74daa542170c4d008f7 HOOK_SERVER_PORT=8999 /usr/local/bin/resty-auto-ssl/dehydrated --cron --accept-terms --no-lock --domain 179.232.255.135 --challenge http-01 --config /etc/resty-auto-ssl/letsencrypt/config --hook /usr/local/bin/resty-auto-ssl/letsencrypt_hooks status: 256 out: # INFO: Using main config file /etc/resty-auto-ssl/letsencrypt/config
startup_hook
Processing 179.232.255.135
 + Signing domains...
 + Generating private key...
 + Generating signing request...
 + Requesting new certificate order from CA...
 err:   + ERROR: An error occurred while sending post-request to https://acme-v02.api.letsencrypt.org/acme/new-order (Status 400)

Details:
HTTP/2 400 
server: nginx
date: Sat, 18 Jan 2025 06:04:42 GMT
content-type: application/problem+json
content-length: 226
boulder-requester: 2176686025
cache-control: public, max-age=0, no-cache
link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
replay-nonce: Ynw3DK8ZPH4y7G-bW1ME-NNx3jzsp0g9AtsZeecA5aWW-zz7jn8

{
  "type": "urn:ietf:params:acme:error:rejectedIdentifier",
  "detail": "Invalid identifiers requested :: Cannot issue for \"179.232.255.135\": The ACME server can not issue a certificate for an IP address",
  "status": 400
}

, context: ssl_certificate_by_lua*, client: 172.27.80.1, server: 0.0.0.0:443
2025/01/18 03:04:42 [error] 277#277: *46 [lua] ssl_certificate.lua:97: issue_cert(): auto-ssl: issuing new certificate failed: dehydrated failure, context: ssl_certificate_by_lua*, client: 172.27.80.1, server: 0.0.0.0:443
2025/01/18 03:04:42 [error] 277#277: *46 [lua] ssl_certificate.lua:291: auto-ssl: could not get certificate for 179.232.255.135 - using fallback - failed to get or issue certificate, context: ssl_certificate_by_lua*, client: 172.27.80.1, server: 0.0.0.0:443
